libs/domain/picking/src/components/customer-note/customer-note.component.ts

```diff
@@ -59,7 +59,7 @@ export class CustomerNoteComponent extends PickingListMixin(LitElement) {
         </button>
       </oryx-button>
       <oryx-picking-in-progress-modal
-        @oryx.close=${this.closePickingInProgressModal}
+        @oryx.close-button=${this.closePickingInProgressModal}
```

Comment:
Choose, please, more convenient name for the event. `oryx.back`, `oryx.navigate-back`, ...
It should be something related to action